src/Passes/Source/RuleTransformationPass/Configuration.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+    /// Configuration class for the RuleTransformation pass.
+    class RuleTransformationPassConfiguration
+    {
+      public:
+        // Setup and construction
+        //
+
+        /// Setup function that attached the configuration to the ConfigurationManager.
+        void setup(ConfigurationManager& config);
+
+        /// Creates a configuration where all functionality is disabled.
+        static RuleTransformationPassConfiguration disable();
+
+        // Configuration classes
+        //
+
+        /// Testing whether this is the default configuration.
+        bool isDisabled() const;
+
+        /// Tests whether all functionality is disabled for this component.
+        bool isDefault() const;
```

Comment:
Did you mean to change places of the comments for these 2 functions? (line 27 and line 30)

Reply:
Indeed, thanks!